scaffolds/cpp/README.md

```diff
@@ -0,0 +1,55 @@
+# C++ Scaffold for Code Retreat
+
+This code scaffold uses [CxxTest](http://cxxtest.com). We recommend using the
+docker image [euranova/cxxtest](https://github.com/euranova/docker-cxxtest), but
+installing `cxxtest` with the package manager is also an option.
```

Comment:
a nice toolchain would check for you the presence of `cxxtest` and/or install it for you (e.g. configure.ac or cmake)

Reply:
Agreed. We went with a docker image instead to avoid having to install anything.

I also have to admit that, unlike you, I've never set up anything with `cmake`, `configure` or `automake`, so it's clearly not my go-to solution. But I'd love to do it once.